tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/LiferayCore.java

```diff
@@ -280,25 +280,37 @@ public void stop(BundleContext context) throws Exception {
 
 		ILiferayProject liferayProject = projectCache.get(projectCacheKey);
 
-		//Stale project should be removed in the first time
-
 		if ((liferayProject != null) && liferayProject.isStale()) {
 			liferayProject = null;
```

Comment:
Hey @gamerson,  this line should be after removing key.

Reply:
Sorry, i means you are removing the null `liferayProject` , if you set it to null first , right?